plugins/husky-npm/package.json

```diff
@@ -12,6 +12,9 @@
   "dependencies": {
     "@dotcom-tool-kit/husky-hook": "file:../../lib/husky-hook"
   },
+  "peerDependencies": {
+    "husky": "^4.3.8"
```

Comment:
**issue**: We want `peerDependencies` to be as permissive as possible, so older minor/patch versions should be allowed too. In theory, we might also be able to support some other major versions if they read the config in a similar way but annoyingly the migration guides are overwritten for each new major version so the compatibility is not immediately obvious.

```suggestion
    "husky": "4.x"
```